.env.example

```diff
@@ -48,3 +48,13 @@ MIX_PUSHER_APP_CLUSTER="${PUSHER_APP_CLUSTER}"
 MOLVENO_ADMIN_USER=
 MOLVENO_ADMIN_EMAIL=
 MOLVENO_ADMIN_PASSWORD=
+
+# below is the settings you need to use with your own username and password to make mailing work
+# this comes instead of the other mail block in the env file
+MAIL_DRIVER=smtp
+MAIL_HOST=smtp.mailtrap.io
+MAIL_PORT=2525
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_FROM_ADDRESS=from@example.com
+MAIL_FROM_NAME=molveno
```

Comment:
De QUEUE_DRIVER variabele ook nog toevoegen

Reply:
na overleg hebben we besloten dat dit niet hoeft

Reply:
Hint: You can close a conversation as resolved by selecting "Resolve conversation" in the GitHub interface.